mmv1/products/compute/BackendService.yaml

```diff
@@ -1307,3 +1380,313 @@ properties:
           The default value is 1.0.
         default_value: 1.0
         diff_suppress_func: 'suppressWhenDisabled'
+      - !ruby/object:Api::Type::Enum
+        name: optionalMode
+        description: |
+          This field can only be specified if logging is enabled for this backend
+          service. Configures whether all, none or a subset of optional fields
+          should be added to the reported logs. One of [INCLUDE_ALL_OPTIONAL,
+          EXCLUDE_ALL_OPTIONAL, CUSTOM]. Default is EXCLUDE_ALL_OPTIONAL.
+          The possible values are:
+
+          * `CUSTOM`: A subset of optional fields.
+
+          * `EXCLUDE_ALL_OPTIONAL`: None optional fields.
+
+          * `INCLUDE_ALL_OPTIONAL`: All optional fields.
+        values:
+          - :CUSTOM
+          - :EXCLUDE_ALL_OPTIONAL
+          - :INCLUDE_ALL_OPTIONAL
+      - !ruby/object:Api::Type::Array
+        name: optionalFields
+        description: |-
+          This field can only be specified if logging is enabled for this backend
+          service and "logConfig.optionalMode" was set to CUSTOM. Contains a list
+          of optional fields you want to include in the logs. For example:
+          serverInstance, serverGkeDetails.cluster,
+          serverGkeDetails.pod.podNamespace
+        item_type: !ruby/object:Api::Type::String {}
+  - !ruby/object:Api::Type::Array
+    name: serviceBindings
+    description: |-
+      URLs of networkservices.ServiceBinding resources. Can only be set if
+      load balancing scheme is INTERNAL_SELF_MANAGED. If set, lists of
+      backends and health checks must be both empty.
+    item_type: !ruby/object:Api::Type::String {}
+  - !ruby/object:Api::Type::String
+    name: kind
+    description: |-
+      [Output Only] Type of resource. Always compute#backendService for
+      backend services.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: selfLink
+    description: '[Output Only] Server-defined URL for the resource.'
+    output: true
+  - !ruby/object:Api::Type::Array
+    name: usedBy
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: reference
+          required: true
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: metadatas
```

Comment:
The description of this reads like a field that is not supposed to be set by users. Is there a reason to add this?